### DISCUSSION TITLE
Developer Meeting 5.12.

### DISCUSSION THREAD

**Discussion by oxisto** [clouditor/clouditor#1269] [Roadmap]
Let's discuss here which issues we want to tackle before EMERALD.

# Issues

# General Stuff
* Ontology update?
* Unify Resource definitions/structs
* List of issues / project board
* We need a graph database for the knowledge graph. Which one?

**Comment by anatheka** [comment]
Azure SDK versions: "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v3" vs "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v5" 

Do we want to update to v5? Is there a possibility to use dependabot for that?

**Reply by anatheka** [reply]
Currently no todo. We update the versions on the fly. Maybe dependabot is able to update to mayor releases in the future.

**Comment by anatheka** [comment]
Add "SetRaw()" to IsCloudResource interface for an easier testing. So we could check if the raw field is set and remove the content for further tests.

**Reply by anatheka** [reply]
@anatheka add and use for tests.

**Comment by immqu** [comment]
- Clouditor implementations: Core vs EMERALD vs COBALT
- Paper / Konferenzen
- MEDINA learnings for EMERALD & COBALT
- JFs Termine
